ics_space_amplification_goal_test.py

```diff
@@ -0,0 +1,232 @@
+import math
+import time
+
+from longevity_test import LongevityTest
+from sdcm.cluster import SCYLLA_DIR, BaseNode
+from sdcm.sct_events import InfoEvent
+from test_lib.compaction import CompactionStrategy, LOGGER
+
+KB_SIZE = 2 ** 10
+MB_SIZE = KB_SIZE * 1024
+GB_SIZE = MB_SIZE * 1024
+FS_SIZE_METRIC = 'node_filesystem_size_bytes'
+FS_SIZE_METRIC_OLD = 'node_filesystem_size'
+AVAIL_SIZE_METRIC = 'node_filesystem_avail_bytes'
+AVAIL_SIZE_METRIC_OLD = 'node_filesystem_avail'
+
+
+class IcsSpaceAmplificationTest(LongevityTest):
+
+    def _get_used_capacity_gb(self, node):  # pylint: disable=too-many-locals
```

Comment:
all those functions are not needed in other tests ? only in this one ?
why they aren't part of cluster of tester ?

Reply:
no other test is interested with nodes used capacity apparently.

Reply:
I don't understand, you are on one hand telling this code isn't being used by any other test.

But on the other hand telling all the issue I've mentioned below like the not f-strings are cause of it copied from old code.

This suggests there are two test files using those functions, and you just duplicate them from on to other.

Let's move those utility functions to a place they can be shared, if that's on the Cluster object or the Tester object.

Reply:
And handle all the things I've noted, while we at it.

Copying code, and not addressing those comments, just cause it's originated from older code, doesn't seem reasonable to me.

Reply:
previous test files were deleted.